plugins/gatsby-transformer-garden-svg/gatsby-node.js

```diff
@@ -5,9 +5,89 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
+const fetch = require('node-fetch');
 const { createNodeHelpers } = require('gatsby-node-helpers');
 const { optimize } = require('svgo');
 
+const tokens = new Map();
+
+exports.onPreInit = async (_, configOptions) => {
+  const { figmaApiToken, fileId, nodeId } = configOptions;
+  const baseURL = 'https://api.figma.com/v1';
+  const headers = {
+    'Content-Type': 'application/json',
+    'X-FIGMA-TOKEN': figmaApiToken
+  };
+
+  const file = await fetch(`${baseURL}/files/${fileId}/nodes?ids=${nodeId}`, { headers });
+  const { nodes } = await file.json();
+  const frame = nodes[Object.keys(nodes)[0]];
+  const { children } = frame.document;
+
+  for (const node of children) {
+    if (node.type === 'COMPONENT') {
+      // eslint-disable-next-line prefer-named-capture-group
+      const name = node.name.replace(/\b(\s+token)\b/u, '').toLowerCase();
```

Comment:
Prefer adding the non-capturing group `?:` notation rather than disabling the eslint rule

Reply:
I'll definitely change it so it uses `?:` instead of `\b`.